python/ray/experimental/raysort/sortlib.py

```diff
@@ -21,7 +21,9 @@ def sort_and_partition(part: np.ndarray,
     return blocks
 
 
-def merge_partitions(blocks: List[np.ndarray],
-                     _n: int) -> Iterable[memoryview]:
+def merge_partitions(
+        num_blocks: int, _n: int,
+        get_block: Callable[[int, int], np.ndarray]) -> Iterable[memoryview]:
+    blocks = [get_block(i, 0) for i in range(num_blocks)]
```

Comment:
Is this doing an actual merge? It looks like it's just returning the blocks again?

Reply:
No, this is a "fake" implementation. The actual merge is implemented in C++ which needs a build step. Should I commit it? (maybe in another PR)

Reply:
Ah I see, I got confused because there was also the `dummy_merge` function. Could we replace the `dummy_merge` implementation with this one (this one looks simpler) and throw a NotImplementedError when the user passes the flag for an actual merge?